BUILDING.md

```diff
@@ -0,0 +1,125 @@
+# From zero to deb packages
```

Comment:
Can you move this file to `debian/README.ubuntu-docker` instead?

Reply:
Sure. I am concerned that not having any standard GitHub documentation means that the probability of people asking how to build will stay high. The truth is, few folks know about all those Debian packaging conventions, they are like Bazaar: nobody neither knows how to use it nor wants to use it hehe.

How about writing a top level `README.md` that references `debian/README.ubuntu-docker`?